homeassistant/components/ffmpeg/__init__.py

```diff
@@ -90,6 +90,14 @@ async def async_service_handle(service):
     return True
 
 
+@bind_hass
+def async_get_ffmpeg_manager(hass: HomeAssistant) -> FFmpegManager:
```

Comment:
This doesn't need to be run in the event loop.
```suggestion
def get_ffmpeg_manager(hass: HomeAssistant) -> FFmpegManager:
```